src/request-pipeline/destination-request/index.js

```diff
@@ -125,6 +126,9 @@ export default class DestinationRequest extends EventEmitter {
     }
 
     _onError (err) {
+        if (err.message && SOCKET_HANG_UP_ERR_RE.test(err.message))
```

Comment:
1)Move  `err.message && SOCKET_HANG_UP_ERR_RE.test(err.message)` to `_isSocketHungUpErr(err)` function

2)Move to on module level regular expressions from body of  the`_isDNSErr` function